integration-libs/cds/src/profiletag/services/profiletag-event.service.ts

```diff
@@ -104,7 +104,17 @@ export class ProfileTagEventService {
     this.exposeConfig(newConfig);
   }
 
+  /*
+   * Checks if the script with the given source exists in the document or not.
+   */
+  private isScriptLoaded(scriptSource: string): boolean {
+    return document.querySelectorAll('[src="' + scriptSource + '"]').length > 0;
```

Comment:
Let's not use the document object ootb. As done in other places in this file, you should access it through `this.winRef` 

You can use text literals (\` \` - backtick) for string interpolation (${ })

you can use querySelectorAll, if you'd like to use the length logic

```suggestion
    return !!this.winRef.document.querySelector(`script[src="${source}"]`);
```

Reply:
Great point! addressed 👍